### PR TITLE
Add additional parameters to http_check

### PR DESCRIPTION
**Pull request by asenci**
Add the check_hostname and ssl_server_name parameters
to the http_check integration

### CODE REVIEW

File: manifests/integrations/http_check.pp [hunk 1]
```diff
@@ -217,6 +219,8 @@
       'check_certificate_expiration' => $check_certificate_expiration,
       'days_warning'                 => $days_warning,
       'days_critical'                => $days_critical,
+      'check_hostname'                => $check_hostname,
```
Comment (albertvaka) [inline manifests/integrations/http_check.pp]: Can you align the rocket arrows on the right?

File: manifests/integrations/http_check.pp [hunk 2]
```diff
@@ -185,6 +185,8 @@
   $check_certificate_expiration = true,
   $days_warning = undef,
   $days_critical = undef,
+  $check_hostname = undef,
```
Comment (albertvaka) [inline manifests/integrations/http_check.pp]: Please use typed arguments. Although we don't have types everywhere  yet, it's preferable to add them for new arguments.

File: manifests/integrations/http_check.pp (hunk 1 above)
Comment (asenci) [reply manifests/integrations/http_check.pp]: Done 👍

File: manifests/integrations/http_check.pp (hunk 2 above)
Comment (asenci) [reply manifests/integrations/http_check.pp]: Done 👍

**Comment by albertvaka** [comment]
Looks good to me! Thanks again @asenci :)

**Comment by albertvaka** [comment]
This just went out in version 3.1.0.

**Pull request merged**